src/SignalR/clients/ts/signalr/src/FetchHttpClient.ts

```diff
@@ -97,6 +124,15 @@ export class FetchHttpClient extends HttpClient {
             payload,
         );
     }
+
+    public getCookieString(url: string): string {
```

Comment:
What's this for?

Reply:
It's the way we apply cookies to WebSockets and SSE

Reply:
So where do we call it? I don't see it in the diff anywhere and it looks new.

Reply:
It isn't new. It just has a [default implementation](https://github.com/dotnet/aspnetcore/blob/e00a7c4d52107e8090d06d1345920ef3e39e3233/src/SignalR/clients/ts/signalr/src/HttpClient.ts#L163) and since the `FetchHttpClient` wasn't used in Node before it wasn't needed. But now it is.

One [usage](https://github.com/dotnet/aspnetcore/blob/e00a7c4d52107e8090d06d1345920ef3e39e3233/src/SignalR/clients/ts/signalr/src/ServerSentEventsTransport.ts#L66).

Reply:
Ah riiiiight. Ok, thanks for clearing that up. Just saw a new public API and was confused. There's a good reason C# has the `override` keyword ;P.